torchprime/metrics/mfu.py

```diff
@@ -137,6 +137,7 @@ def calculate_tflops_training_per_device(config: Config, log=True):
 def compute_mfu(
   config: dict,
   batch_size: int,
+  torch_dtype: str,
```

Comment:
nit: probably you can just default it to bfloat16, so there is no need to update tests